docs/industrial_iot_deployment.md

```diff
@@ -0,0 +1,327 @@
+
+    ``` bash
+    [10:46:35 ERR] Failed to deploy Industrial IoT solution.
+    System.AggregateException: One or more errors occurred. (AADSTS65001: The user or administrator has not consented to use the application with ID 'fb2ca262-60d8-4167-ac33-1998d6d5c50b' named 'AzureIndustrialIoTIAI'. Send an interactive authorization request for this user and resource.
+    Trace ID: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+    Correlation ID: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+    Timestamp: 2019-11-11 09:46:35Z)
+    ...
+    ```
+
+    If you encounter this go to [Granting Admin Consent](#granting-admin-consent) to resolve this issue and then try to run the application again.
+
+6. Provide a name for the AAD applications that will be registered. You will be asked for one name, but 3 applications will be registered:
+
+    * one with `-services` suffix added to provided name
+    * one with `-clients` suffix added to provided name
+    * one with `-aks` suffix added to provided name
+
+    ``` bash
+    Please provide a name for the AAD application to register:
+    kkTestApp51
+    ```
+
+7. Select a Subscription within your Azure Account that will be used for creating Azure resources. If just one Subscription exists in your Account, then it will be listed and application will proceed further, otherwise you have to select Subscription from the list of available Subscriptions:
+
+    ``` bash
+    The following subscription will be used:
+    SubscriptionId: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX, DisplayName: Visual Studio Enterprise
+    ```
+
+8. Select Resource Group. You would be presented with options of either using an existing Resource Group or creating a new onw. If you choose to use an existing one, you will be presented with a list of all Resource Group within the Subscription to choose from. Otherwise, you would be asked to select a Region for the new Resource Group and provide a name for it. Output for the latter case is bellow:
+
+    ``` bash
+    Do you want to create a new ResourceGroup or use an existing one ? Please select N[new] or E[existing]
+    n
+    Please select region where resource group will be created.
+
+    Available regions:
+    0: eastus2
+    1: westus2
+    2: northeurope
+    3: westeurope
+    4: canadacentral
+    5: centralindia
+    6: southeastasia
+    Select a region:
+    3
+    Select resource group name, press Enter to use 'kkTestApp51':
+
+    [13:36:20 INF] Creating Resource Group: kkTestApp51 ...
+    [13:36:21 INF] Created Resource Group: kkTestApp51
+    ```
+
+9. Now the application will perform application registration, Azure resource deployment and deployment of back-end microservices to AKS cluster. This will take around 15 to 20 minutes depending on Azure Region. Along the way, it will log what Azure resources are created. Sample log looks like this:
+
+    ``` bash
+    [13:36:21 INF] Registering resource providers ...
```

Comment:
Do we need to show log lines in README?

Reply:
I decided to do this to show what to expect. I do not have a spinning cursor, or anything like that to show progress, so it might look like program is stuck at times.

Reply:
Cool.